pkg/services/annotations/annotations.go

```diff
@@ -145,3 +145,17 @@ type ItemDTO struct {
 	AvatarUrl   string           `json:"avatarUrl"`
 	Data        *simplejson.Json `json:"data"`
 }
+
+type annotationType int
+
+const (
+	Global annotationType = iota
+	Local
+)
+
+func(annotation *ItemDTO) GetType() annotationType {
+	if annotation.DashboardId != 0 {
+		return Local
+	}
+	return Global
+}
```

Comment:
That is a good point. We started talking about `local` annotations but because it is the oposite for `global` and that term already existed before in code and documentation. The term `local` to refer to annotations related to dashboards does not have much sense. Probably `dashboardAnnotation` is more accurate 🤔.

Reply:
Just for future reference - we added a poll for annotation types, and decided on `annotations:type:dashboard` and `annotations:type:organization`. Thanks for bringing this up!